TrackerConditions/src/TrackerStatusMaker.cc

```diff
@@ -59,11 +59,25 @@ namespace mu2e {
     for (auto const& row : tpls_p->rows()) estatus.insert(TrackerElementStatus(row.id(),tpls_p->sidMask(),row.status()));
     for (auto const& row : tpas_p->rows()) estatus.insert(TrackerElementStatus(row.id(),tpas_p->sidMask(),row.status()));
     for (auto const& row : tssl_p->rows()) estatus.insert(TrackerElementStatus(row.id(),tssl_p->sidMask(),row.status()));
-    for (auto const& row : tsss_p->rows()) estatus.insert(TrackerElementStatus(row.id(),tsss_p->sidMask(),row.status()));
+    unsigned ntotal = tpls_p->rows().size() + tpas_p->rows().size() + tssl_p->rows().size();
+    // tsss can have duplicate entries from tssl
+    for (auto const& row : tsss_p->rows()){
+      auto elem = TrackerElementStatus(row.id(),tsss_p->sidMask(),row.status());
+      auto iter = estatus.find(elem);
+      if (iter == estatus.end()){
+        estatus.insert(elem);
+        ntotal++;
+      }else{
+        TrackerElementStatus temp = *iter;;
+        estatus.erase(iter);
```

Comment:
Why can't you just do iter->status_.merge(elem.status_)?
I also think the ntotal test is no longer sensible and should be removed.

Reply:
I was having some const issue here. Unless we change the comparison operator and allow all the other tables to merge duplicate rows ntotal still makes sense (checks that you don't set status of a panel/plane twice since that won't work)

Reply:
Right, I remember now: set won't allow you to modify an element as that might change the sort, invalidating its internal structure.  Remove and replace works.  The alternative is to use something besides set as estat_t.  But for now lets use your solution.